packages/opentelemetry-instrumentation-openai/opentelemetry/instrumentation/openai/shared/__init__.py

```diff
@@ -205,3 +229,29 @@ def model_as_dict(model):
         return model_as_dict(model.parse())
     else:
         return model
+
+
+def should_record_stream_token_usage():
```

Comment:
Let's put it as an optional parameter to the `__init__` of the instrumentation (I'd default it to `true`)? I think it's cleaner (and I want to move the `should_trace_content` to be like that as well)

Reply:
You mean move it to `Traceloop.init()`'s arg, right?

Reply:
No, the instrumentation init. Then traceloop sdk can pass it yes